### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by blinkseb**
Note: based on #41 

---

Various updates to various scripts. Most notable is 5154bdf which add support for multiple datasets matching a given sample name. Useful to add all signal samples at once, or all data. A warning is shown on the screen in case of it's not really wanted by the user.

@BrieucF Can you check the changes on `basePlotter.py`?  	733b97a

plotIt related part depends on https://github.com/cp3-llbb/plotIt/pull/80

### CODE REVIEW

**Comment by blinkseb** [comment]
PR rebased, only remains Brieuc's comment. It seems to have disappeared, so:

> Lets profit from this PR to correct the logic. I think the correct way to go here is 
> 
> ```
> (({0}.isElMu || {0}.isMuEl) && (hh_elmu_fire_trigger_cut || hh_muel_fire_trigger_cut || runOnMC) && (runOnElMu || runOnMC) && {1}.M() > 12).
> ```

File: plotIt_hh/listHistos.py [hunk 1]
```diff
@@ -54,6 +74,10 @@
 for key in keys:
     if key.GetName() not in alreadyIn and not "__" in key.GetName():
 
+        # Keep only histograms
+        if not key.ReadObj().InheritsFrom("TH1"):
```
Comment (swertz) [inline plotIt_hh/listHistos.py:78]: If you do this you can remove https://github.com/cp3-llbb/HHTools/pull/43/files#diff-19e06715e593fe9eea6306b25e9af506R88 ;)

Comment (blinkseb) [reply plotIt_hh/listHistos.py:78]: `TH2` inherits from `TH1` ;)

**Comment by blinkseb** [comment]
Cut strings for categories fixed

**Pull request merged**